Add valid signatures for jwt_vc_json section (#403)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

<{{examples/credential_response_jwt_vc_json.txt}}

> The following is a non-normative example of a dereferenced document for the Issuer HTTP URL identifier.

Reviewer: Maybe something like this if I've understood what is meant correctly:

```suggestion
The following is the dereferenced document for the Issuer HTTP URL identifier that matches the Credential in the above example:
```

Author: Thanks
